cmake/FindHelper.cmake

```diff
@@ -20,9 +20,9 @@ macro(FIND_HELPER prefix pkg_name header lib)
         set(${prefix}_FOUND TRUE)
     else()
         # use pkg-config if available to set find_path and find_library hints
-        if(NOT WIN32)
+        #if(NOT WIN32)
             FIND_PACKAGE(PkgConfig)
-        endif()
+        #endif()
```

Comment:
It doesn't hurt to try to use `pkg-config` on Windows, I think. It's not required, so if it's not found nothing happens. MSYS2 uses `pkg-config`.